### PR TITLE
parse bitrise yml

### PR DESCRIPTION
**Pull request by trapacska**

### CODE REVIEW

File: phases/bitrise_yml.go [hunk 1]
```diff
@@ -1,9 +1,134 @@
...
+		workflows = append(workflows, workflow)
+	}
+
+	(&option{
+		title:        "Which workflow do you want to run in the first build?",
+		valueOptions: workflows,
+		action: func(answer string) *option {
+			firstWorkflow = answer
+			return nil
+		}}).run()
```
Comment (lszucs) [inline phases/bitrise_yml.go]: [note] could be formatted to be more clear:

```
    }
}).run()
```

**Comment by trapacska** [comment]
I think it might worth to print command calls, and also validate the given inputs before jumping to a next phase.

File: phases/bitrise_yml.go [hunk 2]
```diff
@@ -1,9 +1,134 @@
...
+		}
+	}
+
+	var bitriseYML models.BitriseDataModel
+	if parseErr := yaml.NewDecoder(bitriseYMLFile).Decode(&bitriseYML); parseErr != nil {
+		return "", "", fmt.Errorf("failed to parse "+bitriseYMLPath+", error: %s", parseErr)
+	}
+
+	var workflows []string
+	for workflow := range bitriseYML.Workflows {
```
Comment (lszucs) [inline phases/bitrise_yml.go]: What is the purpose of the manual copying of the array? Why not use `birtiseYML.Workflows` directly?

Comment (trapacska) [reply phases/bitrise_yml.go]: valueOptions: workflows waiting for a []string but bitriseYML.Workflows is a map

**Pull request merged**